package/setup.py

```diff
@@ -329,9 +329,24 @@ def extensions(config):
     util = MDAExtension('lib.formats.cython_util',
                         sources=['MDAnalysis/lib/formats/cython_util' + source_suffix],
                         include_dirs=include_dirs)
-
+    encore_utils = MDAExtension('analysis.encore.cutils',
```

Comment:
This is still an issue. Please fix it.  I don't know why github doesn't show it on the overview page anymore.